docs/source/collect/certificates.md

```diff
@@ -0,0 +1,105 @@
+---
+title: TLS Certificates
+description: Collect Certificate Chain information from Kubernetes ConfigMaps and Secrets in collected output
+---
+
+The `certificates` collector can be used to gather information about the TLS certificates from Kubernetes ConfigMaps and Secrets. This collector can be used multiple times, referencing different Secrets and ConfigMaps.
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `certificates` collector accepts the following parameters:
+
+##### `name` (Required)
+
+The name of the ConfigMap or Secret.
+
```

Comment:
I suggest we remove this parameter both here and in the [code](https://github.com/replicatedhq/troubleshoot/blob/8fb417a8967bce1a770a4c2777129bd573768aff/pkg/apis/troubleshoot/v1beta2/collector_shared.go#L221). Its not used by the collector and just creates confusion of when to use it. We have `collectorName` which all collectors can define.